libs/asset-viewer/src/lib/components/asset-viewer-page/asset-viewer-page.component.ts

```diff
@@ -70,6 +70,20 @@ export class AssetViewerPageComponent implements AfterViewInit, OnDestroy {
   private _cd = inject(ChangeDetectorRef);
   private _ngZone = inject(NgZone);
 
+  public isLoading$ = combineLatest([
+    this._store.select(selectIsMapInitialized),
+    this._store.select(selectFilterLoadingState),
+    this._store.select(selectSearchLoadingState),
+    this._store.select(selectAssetDetailLoadingState),
+  ]).pipe(
+    map(
```

Comment:
No need for a pipe.map, there is a result selector convenience function in combinelatest:
```
  public isLoading$ = combineLatest([
    this._store.select(selectIsMapInitialized),
    this._store.select(selectFilterLoadingState),
    this._store.select(selectSearchLoadingState),
    this._store.select(selectAssetDetailLoadingState),
  ], (isMapInitialized, filterLoadingState, searchLoadingState, detailLoadingState) =>
    !isMapInitialized ||
    filterLoadingState === LoadingState.Loading ||
    searchLoadingState === LoadingState.Loading ||
    detailLoadingState === LoadingState.Loading );
```